python/py-pydantic/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pydantic
+version             1.5.1
+categories-append   devel
+platforms           darwin
+supported_archs     noarch
+license             MIT
+
+python.versions     36 37 38
```

Comment:
I would prefer not to add a `py36` subport for new ports, that also avoids adding the other port in this PR